tests/unit/bokeh/plotting/test__renderer.py

```diff
@@ -84,6 +84,31 @@ def test_override_defaults(self) -> None:
         assert ca["fill_color"] == "white"
         assert set(ca) == { "fill_color", "hatch_color", "line_color", "fill_alpha", "hatch_alpha", "line_alpha" }
 
+class Test__make_glyph:
```

Comment:
```suggestion
class Test_make_glyph:
```
current name would be appropriate for `_make_glyph`